src/ert/storage/local_ensemble.py

```diff
@@ -655,7 +595,7 @@ def save_gen_kw(  # pylint: disable=R0913
         parameter_name: str,
         parameter_keys: List[str],
         realizations: List[int],
-        data: npt.NDArray[np.float64],
+        data: npt.NDArray[np.double],
```

Comment:
Not really part of this PR, but you can safely remove "# pylint: disable=R0913" next to `def save_gen_kw` as the function does not have "too-many-arguments".

Reply:
removed it